app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt

```diff
@@ -138,10 +138,14 @@ object JavaScriptActionHandler {
     }
 
     @JvmStatic
-    fun mobileWebChromeShim(): String {
+    fun mobileWebChromeShim(isMainPage: Boolean): String {
+        var extraHeaderCSS = ""
+        if (isMainPage) {
```

Comment:
Any reason this is only for the main page?

Reply:
No. Removed it, thanks!